Sources/PerformanceMetrics/Trackers/PerformanceMetrics+MultiTracker.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+
+public extension PerformanceMetrics {
+
+    /// A performance metrics tracker that forwards performance measuring events to multiple trackers, while not doing
+    /// any tracking on its own.
+    public class MultiTracker: PerformanceMetricsTracker {
+
+        /// The configured sub trackers.
+        public let trackers: [PerformanceMetricsTracker]
+
+        /// The tracker's tokenizer.
+        private let tokenizer = Tokenizer<Tag>()
+
+        /// The tracker's token dictionary, containing the mapping between internal and sub trackers' tokens.
+        private let tokens = Atomic<[Token<Tag> : [Token<Tag>]]>([:])
```

Comment:
Are we following the Mindera style guide for Alicerce? If so.... `= [:]` is preferred over `([:])` 💄

(sorry I know it was like this before too.... but just thought I would mention it while I saw it. Feel free to ignore if you want to 😄)

Reply:
As reference: https://github.com/Mindera/swift-style-guide

Reply:
Yes, we try to follow our style guide, even though unfortunately it's not fully consistent yet throughout the project 😇 

Regarding this particular case, remember that `Atomic` is not a dictionary, but a wrapper over a dictionary (in this case). It's also not `ExpressibleByDictionaryLiteral`, so I don't think we can instantiate it like you mention 😆 (I just tried to add that conformance as an experiment, but failed miserably 😅)

Reply:
Ahh of course, yeah that makes sense 👍 

Thanks 😄